src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/HandleWithdrawalRequest.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Newtonsoft.Json;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.Withdrawals;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.Withdrawals.Types;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities
+{
+    public class HandleWithdrawalRequest
+    {
+        private readonly IWithdrawalService _withdrawalService;
+
+        public HandleWithdrawalRequest(IWithdrawalService withdrawalService)
+        {
+            _withdrawalService = withdrawalService;
+        }
+
+        [FunctionName("HttpTriggerWithdrawalRequest")]
+        public async Task<IActionResult> RunHttp([HttpTrigger(AuthorizationLevel.Function, "POST", Route = "withdraw")] HttpRequestMessage request)
+        {
+            try
+            {
+                var withdrawRequest = JsonConvert.DeserializeObject<WithdrawRequest>(await request.Content.ReadAsStringAsync());
+                await _withdrawalService.Withdraw(withdrawRequest);
+            }
+            catch(ArgumentException ex)
+            {
+                return new ContentResult()
+                {
+                    StatusCode = (int)HttpStatusCode.BadRequest,
+                    ContentType = "application/json",
+                    Content = JsonConvert.SerializeObject(new { 
+                        ex.ParamName, 
+                        ex.Message, 
+                        Example = new  
+                        { 
+                            Type = "Employer",
+                            AccountLegalEntityId = 1234,
```

Comment:
Would it be a good idea to include the account legal entity id and ULN from the request in the error response?

Reply:
The Example is just what is expected.  If there wasn't a a ULN then it wouldn't be able to be used but the error message would show that.